data/mods/Magiclysm/items/black_dragon_items.json

```diff
@@ -433,7 +443,6 @@
     "warmth": 9,
     "material_thickness": 4,
     "environmental_protection": 3,
-    "material": [ "black_dragon_hide" ],
```

Comment:
why does this not need material?

Reply:
That line is for the XL dragon hide backpack, so it should be _inheriting_ its material from the regular D-hide backpack.

Reply:
That’s what I thought but if materials inherit Im confused as to how it hasn’t been breaking the game previously.

Reply:
Well, inherited data can be sort of _written over_ like this in case you want to inherit some other data but not the material, but since the material is the same in this case it's just a redundant declaration.